src/js/components/Grommet/__tests__/Grommet-test.js

```diff
@@ -129,9 +122,11 @@ describe('Grommet', () => {
     /* eslint-enable max-len */
   ].forEach(ua => {
     test(`ssr rendering ${ua.substring(0, 25)}`, () => {
-      const component = renderer.create(<SSRTester ua={ua} />);
-      expect(component.toJSON()).toMatchSnapshot();
-      component.unmount();
+      act(() => {
+        const { container } = render(<SSRTester ua={ua} />);
+
+        expect(container.firstChild).toMatchSnapshot();
+      });
```

Comment:
Hey, @jcfilben. I just improved the tests.

As I mentioned in the ResponsiveContext tests when we render with testing-library render it triggers the `useEffect` before starting the assertions. And as I also mentioned the `useEffect` changes the `size` based on the `document.body.clientWidth`. 

But in these specific tests, we want to test the `userAgent` so I decided to wrap the `render` in an `act` and test the snapshots inside this `act` so it doesn't trigger the `useEffect` which means we are testing the `userAgent`.

What do you think?

Reply:
Nice solution, looks good to me